examples/zkapps/08-custom-tokens/src/main.ts

```diff
@@ -9,131 +9,120 @@ import {
   Signature,
 } from 'snarkyjs';
 
-(async function main() {
-  await isReady;
+await isReady;
 
-  console.log('SnarkyJS loaded');
+console.log('SnarkyJS loaded');
 
-  const proofsEnabled = false;
-  const Local = Mina.LocalBlockchain({ proofsEnabled });
-  Mina.setActiveInstance(Local);
-  const deployerAccount = Local.testAccounts[0].privateKey;
+const proofsEnabled = false;
+const Local = Mina.LocalBlockchain({ proofsEnabled });
+Mina.setActiveInstance(Local);
+const deployerAccount = Local.testAccounts[0].privateKey;
+// ----------------------------------------------------
 
-  // ----------------------------------------------------
+const zkAppPrivateKey = PrivateKey.random();
+const zkAppAddress = zkAppPrivateKey.toPublicKey();
 
-  const zkAppPrivateKey = PrivateKey.random();
-  const zkAppAddress = zkAppPrivateKey.toPublicKey();
+console.log('compiling...');
 
-  console.log('compiling...');
+let { verificationKey } = await BasicTokenContract.compile();
 
-  let verificationKey: any;
-  if (proofsEnabled) {
-    ({ verificationKey } = await BasicTokenContract.compile());
-  }
+console.log('compiled');
 
-  console.log('compiled');
+// ----------------------------------------------------
 
-  // ----------------------------------------------------
-
-  console.log('deploying...');
-  const contract = new BasicTokenContract(zkAppAddress);
-  const deploy_txn = await Mina.transaction(deployerAccount, () => {
-    AccountUpdate.fundNewAccount(deployerAccount);
-    contract.deploy({ verificationKey, zkappKey: zkAppPrivateKey });
-  });
-  await deploy_txn.prove();
-  await deploy_txn.send();
+console.log('deploying...');
+const contract = new BasicTokenContract(zkAppAddress);
+const deploy_txn = await Mina.transaction(deployerAccount.toPublicKey(), () => {
+  AccountUpdate.fundNewAccount(deployerAccount.toPublicKey());
+  contract.deploy({ verificationKey, zkappKey: zkAppPrivateKey });
+});
+await deploy_txn.prove();
+deploy_txn.sign([deployerAccount]);
```

Comment:
nit: we can combine the .sign() and .send in one line

`await deploy_txn.sign([deployerAccount]).send();`